src/v2/Components/v2/ArtworkFilter/ArtworkFilters/index.tsx

```diff
@@ -6,7 +6,7 @@ import { GalleryFilter } from "./GalleryFilter"
 import { InstitutionFilter } from "./InstitutionFilter"
 import { MediumFilter } from "./MediumFilter"
 import { PriceRangeFilter } from "./PriceRangeFilter"
-import { SizeRangeFilter } from "./SizeRangeFilter"
+import { SizeFilter } from "v2/Apps/Artist/Routes/AuctionResults/Components/AuctionFilters/SizeFilter"
```

Comment:
I'm surprised that this can just be plugged in since the underlying component seems to depend on the `AuctionResultsFilterContext` 😮.

I'm not convinced we need to re-use that exact component here, but if we _do_, we should also move it to a shared `Components` folder! Otherwise it won't be clear that it's used in multiple contexts.

Reply:
I am also a bit surprised, and I was waiting for the mp changes to be there at least on staging so i can actually try to use this and see if it works or no 😬. This PR was just trying out things while the gravity and mp changes are finished. I thought of moving it, but I didnt see an obvious shared place for components, so I tried to make the smallest change.

I will move it in a shared place, and I added `[WIP]` in the title so while its not tested 👍.